icinga2x/Vagrantfile

```diff
@@ -36,7 +37,6 @@ nodes = { 'icinga2' => {
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   nodes.each_pair do |name, options|
     config.vm.define name do |node_config|
-      node_config.vm.box = options[:box_virtualbox]
```

Comment:
No actually I get some errors with it. Basically for openstack (I think, might be wrong) I don't want to download a box. Anyway, I can still remove the box option for openstack.

https://github.com/martbhell/icinga-vagrant/blob/05d44cc3e94155ce72a00d1a1dbe91faf1eb0535/icinga2x/Vagrantfile#L89 has an override so that sets it for virtualbox provider. Isn't that enough?